app/eventyay/orga/templates/orga/includes/dashboard_kpi_cards.html

```
@@ -0,0 +1,134 @@
+{% load i18n %}
+<section class="td-section" aria-labelledby="td-overview-title">
+    <div class="td-section-heading">
+        <h2 id="td-overview-title" class="td-section-title">
+            <span class="fa fa-tachometer" aria-hidden="true"></span>
+            {% trans "At a glance" %}
+        </h2>
+    </div>
+
+    <div class="td-kpi-grid">
+        <a href="{{ request.event.orga_urls.live }}" class="td-kpi-card td-status-card {% if at_a_glance.talk_component_status.text == 'Live' %}td-status-card-green{% else %}td-status-card-blue{% endif %}" title="{% trans "Talk component status" %}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)